src/main/resources/js/user_information.js

```diff
@@ -67,7 +73,7 @@ AJS.toInit(function () {
             
             var done = userInformation[i].state === "DONE";
 
-            var enableButton = "<button class='aui-button' id='button"+ userInformation[i].timesheetID + "'>Enable Timesheet</button>";
+            var enableDropdown =  "<select id='active-timesheet" + userInformation[i].timesheetID + "' onchange='getComboA(this)'><option value='ACTIVE'>ACTIVE</option><option value='DISABLED'>DISABLED</option><option value='SHOW'>SHOW</option><option value='selectAction' selected disabled>SELECT ACTION</option></select>";
```

Comment:
Please use components from the Atlassian User Interface (AUI) instead of a self-made selection. You can find more information at https://docs.atlassian.com/aui/8.6.0/docs/dropdown.html.

Example:
```
<button class="aui-button aui-dropdown2-trigger" aria-controls="example-dropdown">
  Options
</button>

<aui-dropdown-menu id="example-dropdown">
    <aui-section label="Links">
        <aui-item-link href="http://atlassian.com">Atlassian</aui-item-link>
        <aui-item-link href="http://github.com">Github</aui-item-link>
    </aui-section>
</aui-dropdown-menu>
```
This will produce a dropdown like you know from the Atlassian user interface:

![Screenshot 2020-04-17 at 14 29 58](https://user-images.githubusercontent.com/6848980/79570579-fcfd8680-80b9-11ea-9b67-c9a52e183a9d.png)